### PR TITLE
Add initial docs

### PR DESCRIPTION
**Pull request by jeremyrsmith**
Also fix python param hints

### CODE REVIEW

**Comment by jeremyrsmith** [comment]
Docs are still WIP

File: docs/01-installation.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+`polynote-dist.tar.gz` file. Unpack the archive:
+
+```
+tar -zxvpf polynote-dist.tar.gz
+cd polynote
+```
+
+## Prerequisites
+- Polynote is currently only tested on Linux and MacOS, using the Chrome browser as a client. We hope to be testing
+  Windows (and other browsers) soon. Feel free to try it on your platform, and be sure to let us know about any issues
```
Comment (jonathanindig) [inline docs/01-installation.md]: Windows?

Comment (jeremyrsmith) [reply docs/01-installation.md]: I'll just edit it to say "other platforms and browsers" so as not to over-commit 😆

Comment (jonathanindig) [reply docs/01-installation.md]: :)

I think Windows support for clients (say, Chrome on Windows) is something we should probably look into. 

Running the actual server on Windows seems more daunting...

**Pull request merged**